contrib/mod_tls.c

```diff
@@ -754,7 +754,6 @@ static size_t tls_data_ticket_appdatasz = 0, tls_data_ticket_appdata_len = 0;
 /* OpenSSL variables */
 static SSL *ctrl_ssl = NULL;
 static SSL_CTX *ssl_ctx = NULL;
-static X509_STORE *tls_crl_store = NULL;
```

Comment:
We no longer need a separate `X509_STORE` object to hold CRLs.  A given `X509_STORE` can hold _both_ trusted CAs _and_ CRLs.  So rather than using a separate object, we will load our CRLs into the _existing_ `X509_STORE` used, in the `SSL_CTX`, for the `TLSCACertificateFile/Path` CAs.